Components/Tests/ComponentsTests/WKSourceEditorFormatterTests.swift

```diff
@@ -766,4 +772,170 @@ final class WKSourceEditorFormatterTests: XCTestCase {
             XCTAssertEqual(base2Attributes[.font] as! UIFont, fonts.baseFont, "Incorrect base formatting")
             XCTAssertEqual(base2Attributes[.foregroundColor] as! UIColor, colors.baseForegroundColor, "Incorrect base formatting")
         }
+
+    func testSubscript() {
+        let string = "Testing. <sub>Subscript.</sub> Testing"
+        let mutAttributedString = NSMutableAttributedString(string: string)
+
+        for formatter in formatters {
+            formatter.addSyntaxHighlighting(to: mutAttributedString, in: NSRange(location: 0, length: string.count))
+        }
+
+        var base1Range = NSRange(location: 0, length: 0)
+        let base1Attributes = mutAttributedString.attributes(at: 0, effectiveRange: &base1Range)
+
+        var subscriptOpenRange = NSRange(location: 0, length: 0)
+        let subscriptOpenAttributes = mutAttributedString.attributes(at: 9, effectiveRange: &subscriptOpenRange)
+
+        var subscriptContentRange = NSRange(location: 0, length: 0)
+        let subscriptContentAttributes = mutAttributedString.attributes(at: 14, effectiveRange: &subscriptContentRange)
+
+        var subscriptCloseRange = NSRange(location: 0, length: 0)
+        let subscriptCloseAttributes = mutAttributedString.attributes(at: 27, effectiveRange: &subscriptCloseRange)
+
+        var base2Range = NSRange(location: 0, length: 0)
+        let base2Attributes = mutAttributedString.attributes(at: 35, effectiveRange: &base2Range)
+
+        // "Testing. "
+        XCTAssertEqual(base1Range.location, 0, "Incorrect base formatting")
+        XCTAssertEqual(base1Range.length, 9, "Incorrect base formatting")
+        XCTAssertEqual(base1Attributes[.font] as! UIFont, fonts.baseFont, "Incorrect base formatting")
+        XCTAssertEqual(base1Attributes[.foregroundColor] as! UIColor, colors.baseForegroundColor, "Incorrect base formatting")
+
+        // "<sub>"
+        XCTAssertEqual(subscriptContentRange.location, 14, "Incorrect strikethrough formatting")
+        XCTAssertEqual(subscriptContentRange.length, 10, "Incorrect strikethrough formatting")
```

Comment:
These two should test `subscriptOpenRange`, location should be 9 with a length of 5.